pyhafas/profile/nvv/requests/journeys.py

```diff
@@ -0,0 +1,13 @@
+from pyhafas.profile import ProfileInterface
+from pyhafas.profile.interfaces.requests.journeys import JourneysRequestInterface
+from pyhafas.profile.base import BaseJourneysRequest
+from pyhafas.types.hafas_response import HafasResponse
+
+
+class NVVJourneysRequest(BaseJourneysRequest, JourneysRequestInterface):
+    def parse_journeys_request(self:ProfileInterface, data: HafasResponse):
+        for jny in data.res["outConL"]:
+            jny["ctxRecon"] = ""
```

Comment:
This is not a good fix, because the ctxRecon (aka journey id) is not missing. It's just in another position.

```suggestion
            jny["ctxRecon"] = jny["recon"]["ctx"]
```